src/pages/index.js

```diff
@@ -5,123 +5,255 @@ import PopupWithForm from "../components/PopupWithForm.js";
 
-const section = new Section(
-  { items: initialCards, renderer: renderCard },
-  ".cards__list"
-);
-section.renderItems();
+api
+  .getProfileInfo()
+  .then((userData) => {
+    userInfo.setUserInfo(userData.name, userData.about);
+    userInfo.setAvatar(userData.avatar);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
 // PopupWithForm
 // PopupWithForm
 
+const newCardPopup = new PopupWithForm({
+  popupSelector: "#add-card-modal",
+  handleFormSubmit: handleAddCardFormSubmit,
+});
+
 const editProfilePopup = new PopupWithForm({
   popupSelector: "#edit-modal",
   handleFormSubmit: handleProfileEditSubmit,
 });
-editProfilePopup.setEventListeners();
 
-const newCardPopup = new PopupWithForm({
-  popupSelector: "#add-card-modal",
-  handleFormSubmit: handleAddCardFormSubmit,
+const avatarPopup = new PopupWithForm({
+  popupSelector: "#profile-avatar-modal",
+  handleFormSubmit: handleAvatarFormSubmit,
 });
-newCardPopup.setEventListeners();
 
 // PopupWithImage
 // PopupWithImage
 
 const newImagePopup = new PopupWithImage("#image-preview-modal");
-newImagePopup.setEventListeners();
 
-// UserInfo
-// UserInfo
+// DeletePopup
+// DeletePopup
+
+const cardDeletePopup = new DeleteConfirm("#delete-card", deleteCardForm);
 
-const userInfo = new UserInfo(profileNameElement, jobElement);
+// Event Listeners
+// Event Listeners
 
-// Validation
-// Validation
+profileFormElement.addEventListener("submit", (event) => {
+  event.preventDefault();
+});
+newImagePopup.setEventListeners();
+cardDeletePopup.setEventListeners();
+editProfilePopup.setEventListeners();
+newCardPopup.setEventListeners();
+avatarPopup.setEventListeners();
+
+// Form Validation
+// Form Validation
 
-const editFormValidator = new FormValidator(settings, profileEditModal);
-const addFormValidator = new FormValidator(settings, addCardModal);
+const editFormValidator = new FormValidator(settings, profileFormElement);
 editFormValidator.enableValidation();
+
+const addFormValidator = new FormValidator(settings, addCardForm);
 addFormValidator.enableValidation();
 
-//Functions
-//Functions
+const avatarFormValidator = new FormValidator(settings, editAvatarForm);
+avatarFormValidator.enableValidation();
 
-function handleProfileEditSubmit(userData) {
-  userInfo.setUserInfo(userData);
-  editProfilePopup.close();
+// Modals
+// Modals
+
+function handleImageClick(name, link) {
+  newImagePopup.open({ name, link });
 }
 
-function prefillProfileData() {
-  const { name, description } = userInfo.getUserInfo();
-  nameInput.value = name;
-  jobInput.value = description;
-  editFormValidator.resetValidation();
+addNewCardButton.addEventListener("click", () => {
+  newCardPopup.open();
+});
+
+editAvatarButton.addEventListener("click", () => {
+  avatarPopup.open();
+});
+
+profileEditBtn.addEventListener("click", (e) => {
   editProfilePopup.open();
+  e.preventDefault();
+  const newUserInfo = userInfo.getUserInfo();
+  newUserInfo.setInputValues();
```

Comment:
* There is an error when I open the profile popup https://snipboard.io/4rs3mn.jpg 


It should be like this

`  editProfilePopup.setInputValues(newUserInfo);
`